python/ray/tests/test_actor.py

```diff
@@ -296,24 +296,6 @@ def f():
     print(ray.get(x_id))  # This should not hang.
 
 
-def test_actor_init_error_propagated(ray_start_regular_shared):
```

Comment:
Hey @lixin-wei we can't break API behavior, this PR must be able to pass existing tests without changes.